backend/tests/unit/services/test_dashboard_update_service.py

```
@@ -369,6 +367,146 @@ async def test_session_archiving_on_completion(self, update_service, mock_broadc
+    @pytest.fixture
+    def update_service(self, mock_broadcaster):
+        """Create service instance."""
+        return DashboardUpdateService(mock_broadcaster)
+    
+    @pytest.mark.asyncio
+    @pytest.mark.unit
+    @pytest.mark.parametrize("terminal_status", [
+        "completed",
+        "failed",
+    ])
+    async def test_terminal_statuses_archive_session(self, update_service, mock_broadcaster, terminal_status):
```

Ruff (0.13.2)

Unused method argument: `mock_broadcaster`

(ARG002)